docs/DeveloperGuide.md

```diff
@@ -1,34 +1,1135 @@
+  * [**Appendices**](#appendices)
+    + [Appendix A: Product Scope](#appendix-a-product-scope)
+    + [Appendix B: User Stories](#appendix-b-user-stories)
+    + [Appendix C: Value proposition - Use cases](#appendix-c-value-proposition---use-cases)
+    + [Appendix D: Non-Functional Requirements](#appendix-d-non-functional-requirements)
+    + [Appendix E: Glossary](#appendix-e-glossary)
+    + [Appendix F: Instructions for Manual Testing](#appendix-f-instructions-for-manual-testing)
+    + [Appendix G: Supported Formats of Date Input](#appendix-g-supported-formats-of-date-input)
 
+## 1. <a id="intro">Introduction</a>
+### 1.1.  <a id="background">Background</a>
+The Schwarzenegger is a desktop command line interface-based app for managing all your needs regarding fitness. With the built-in personal assistant, you are able to track your daily workout and diet sessions based on your profile. If you can type fast, The Schwarzenegger can help you maximise your efficiency for maintaining fitness.
+### 1.2. <a id="purpose">Purpose</a>
+This document contains the specified architecture and software design specifications for the application, The Schwarzenegger. 
+### 1.3. <a id="scope">Scope</a>
+This document describes the software architecture and software design requirements for The Schwarzenegger. This guide is mainly for developers, designers and software engineers that are or going to work on The Schwarzenegger. 
 
-## Product scope
-### Target user profile
+[&#8593; Return to Top](#developer-guide)
 
-{Describe the target user profile}
+## 2. <a id="setting-up">Setting Up</a>
 
-### Value proposition
+### 2.1. <a id="prerequisites">Prerequisites</a>
+1. JDK `11`.
+2. IntelliJ IDEA IDE.
 
-{Describe the value proposition: what problem does it solve?}
+### 2.2. <a id="setting-up-the-project-in-your-computer">Setting up the Project in Your Computer</a>
+1. Fork [this repository](https://github.com/AY2021S1-CS2113T-F11-1/tp), and clone the fork to your computer.
+2. Open IntelliJ (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first).
+3. Set up the correct JDK version for Gradle  
+   1. Click `Configure` > `Structure for New Projects` and then `Project Settings` > `Project` > `Project SDK`.
+   2. If `JDK 11` is listed in the drop down, select it. Otherwise, click `New…` and select the directory where you installed `JDK 11`.
+   3. Click `OK`.
+4. Click `Import Project`.
+5. Locate the `build.gradle` file and select it. Click `OK`.
+6. Click `Open as Project`.
+7. Click `OK` to accept the default settings if prompted. 
 
-## User Stories
+[&#8593; Return to Top](#developer-guide)
+
+## 3. <a id="design">Design</a>
+This section provides a high level overview of our application, The Schwarzenegger.
+### 3.1. <a id="architect">Architecture</a>
+
+![Architecture](pictures/zesong/Architecture.png)
+
+The image above explains the design of the application, The Schwarzenegger. 
```

Comment:
Should it be better if the diagram is resized?
<img width="444" alt="942CABCC-FAA9-47D5-957F-47A7FFF74042" src="https://user-images.githubusercontent.com/57485187/97532108-9a822d00-19f0-11eb-9f14-e5b8224ae29a.png">